integration/insert_compat_test.go

```diff
@@ -23,12 +23,14 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
 
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
 type insertCompatTestCase struct {
-	insert     bson.D                   // required
+	insert     []any                    // required, slice of bson.D to be insert
```

Comment:
If I understand, we use `[]any` here, and not `[]bson.D` because `InsertMany` function wants the `[]interface{}` as an argument. Then shouldn't we use `[]interface{}` here?  If I'm reasoning correctly, using `any` statement here mixes generics with interfaces

Reply:
`any` is an alias for `interface{}` since 1.18. It is not about generics. We should not use `interface{}` anywhere

Reply:
I understand the concept, though I remember we had similar usage of `any` on another issue and we said that we shouldn't do that, but maybe I misunderstood something. If that's correct usage then I'll remember that